Documentation/kubernetes-upgrade.md

```diff
@@ -47,18 +46,15 @@ The kubelet would then restart the pod, and the new image version would be used.
 
 ### Upgrading Remaining Master Node Components (High-Availability)
 
-The kube-controller-manager, kube-scheduler, and kube-podmaster are all also deployed as static pods in `/etc/kubernetes/manifests`. However, in high-availability deployments, the kube-podmaster is responsible for making sure only a single copy of the controller-manager and scheduler are running cluster-wide.
+The kube-controller-manager and kube-scheduler are also deployed as static pods in `/etc/kubernetes/manifests`. In high-availability deployments, kube-controller-manager and kube-scheduler must be used with the leader-elect flag and can be run on all master nodes
```

Comment:
This (and the following sentence) are true for apiserver, scheduler, controller-manager - so we might want to change this verbiage to something that references them all:

> In high-availability deployments, the control-plane components (apiserver, scheduler, and controller-manager) are deployed to all master nodes. Upgrades of these components will require them being updated on each master node.

> The controller-manager and scheduler, while running on all master nodes, will use leader-election so only a single copy is doing work at any given time. When updating these components be aware that updating a single copy may not affect the currently active component (and be sure that all are eventually updated).